inc/gui.class.php

```diff
@@ -472,14 +472,14 @@ public static function options_page() { ?>
 							<?php esc_html_e( 'More', 'antispam-bee' ); ?>
 						</h3>
 						<h6>
-							<?php esc_html_e( 'A few little things', 'antispam-bee' ); ?>
+							<?php esc_html_e( 'Less relevant options', 'antispam-bee' ); ?>
```

Comment:
"Weniger relevant" is vielleicht nicht ganz passend hier, da es dann den Anschein erweckt, dass die Optionen nix mit Antispam Bee zu tun hätten.
Wie wäre es mit `'Miscellaneous options'` um zu sagen dass es Optionen sind die in keine der anderen Kategorien passen.
Oder `'Integration options'`, da es das im Prinzip trifft.